website/sidebars.js

```diff
@@ -341,5 +341,22 @@ module.exports = {
                 },
             ],
         },
+        {
+            label: 'Contributing',
+            type: 'category',
+            collapsed: false,
+            link: {
+                type: 'generated-index',
+                title: 'Developer Guide',
+                description: 'Learn how to contribute to unleash.',
+                slug: '/developer-guide',
+            },
+            items: [
+                'contributing/developer-guide',
+                'contributing/frontend/overview',
+                'contributing/backend/overview',
+                'contributing/ADRs',
```

Comment:
We should list all the contribution docs from the sidebar (as remarked on in a previous comment). However, instead of listing each file individually (as we have done in other parts of the sidebar), I think we should try and autogenerate this section if possible. Refer to the [autogenerated sidebar docs](https://docusaurus.io/docs/sidebar/autogenerated) for more info. Give it a go and see if it produces satisfying results?